internal/lsp/rego/builtins.go

```diff
@@ -2,15 +2,23 @@ package rego
 
 import (
 	"strings"
+	"sync"
 
 	"github.com/open-policy-agent/opa/ast"
 )
 
-var BuiltIns = builtinMap() //nolint:gochecknoglobals
+var BuiltInsLock = &sync.RWMutex{}                          // nolint:gochecknoglobals
+var BuiltIns = builtinMap(ast.CapabilitiesForThisVersion()) //nolint:gochecknoglobals
 
-func builtinMap() map[string]*ast.Builtin {
+func UpdateBuiltins(caps *ast.Capabilities) {
+	BuiltInsLock.Lock()
+	defer BuiltInsLock.Unlock()
+	BuiltIns = builtinMap(caps)
```

Comment:
[nit]: If we add a concurrency-safe update method here, should we perhaps make `Builtins` private (↔️ `builtins`)?

Reply:
b618eb5761a86ddc7420f796a504a3d4b9057fce